ccd-definition/CaseEventToFields/CareSupervision/draftSDO.json

```diff
@@ -58,7 +58,8 @@
     "DisplayContext": "COMPLEX",
     "PageID": "judgeAndLegalAdvisor",
     "PageDisplayOrder": 3,
-    "PageColumnNumber": 1
+    "PageColumnNumber": 1,
+    "PageShowCondition": "sdoRouter=\"SERVICE\" OR useServiceRoute=\"YES\""
```

Comment:
hmm, what about upload path, this page is there as well

Reply:
Doh, of course 👍